scripts/insert_into_appointments.sql

```diff
@@ -1 +1,6 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(date_on, time_on, doctor_id, patient_id)VALUES
+ (DATE '2016-Aug-12',TIME '10:00', 2, 3),
+ (DATE '2016-Aug-17',TIME '13:00'),  3, 4,
+ (DATE '2016-Aug-12',TIME '10:00', ),
+ (DATE '2016-Aug-17',TIME '13:00');
```

Comment:
This wouldn't work here, this would:

```sql
-- insert appointments creating connections between doctors and patients
INSERT INTO appointments(date, time, doctor_id, patient_id)
  SELECT DATE '2017-08-12', TIME '1000',
    (SELECT id FROM doctors WHERE family_name = 'Hilton'),
    (SELECT id FROM patients WHERE family_name = 'Wilcox')
  UNION
  SELECT '2017-08-17', '1300',
    (SELECT id FROM doctors WHERE family_name = 'Hubbard'),
    (SELECT id FROM patients WHERE family_name = 'Talley')
  UNION
  SELECT '2017-08-17', '1000',
    (SELECT id FROM doctors WHERE family_name = 'Hubbard'),
    (SELECT id FROM patients WHERE family_name = 'Wilcox')
  UNION
  SELECT '2017-08-17', '1300',
   (SELECT id FROM doctors WHERE family_name = 'Hilton'),
   (SELECT id FROM patients WHERE family_name = 'Hodge')
;
```